com.unity.netcode.adapter.utp/Editor/MultiplayerWindow.cs

```diff
@@ -0,0 +1,60 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.Netcode.Editor
+{
+    public class MultiplayerWindow : EditorWindow
+    {
+        private const string k_PrefsKeyPrefix = "NetcodeGameObjects";
+
+        [MenuItem("Netcode/NGO Tools")]
+        public static void ShowWindow()
+        {
+            GetWindow<MultiplayerWindow>(false, "NGO Tools Tools", true);
```

Comment:
Minor, but is the duplicated "Tools" here on purpose?

Reply:
no will fix :P